customer-wallet/src/containers/Reviews/test-data/tripadvisor.json

```diff
@@ -0,0 +1,125 @@
+[{
```

Comment:
Is this test data straight out of the apify crawler? Or was some transformation applied?